temp/gitbasics.txt

```diff
@@ -0,0 +1,29 @@
+1.git config --global user.name "username"
+2.git config --global user.email "email"
+3.git init
```

Comment:
Still, the task is not complete.
Example: git init > It creates a .git file, It is used in backend to switch the branch, store the changes in local and remote, etc.